src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -82,51 +82,61 @@
 
               <!-- Business or corporation info -->
               <template v-if="isOrg">
-                <!-- Add firm org using look-up -->
-                <article v-if="showAddFirmOrgComponents && orgPerson.isLookupBusiness" class="org-look-up">
-                  <label class="sub-header">Business or Corporation Look Up</label>
-
-                  <a class="lookup-toggle float-right" @click="toggleLookup()">
-                    Business or Corporation is Unregistered in B.C.
-                  </a>
+                <v-checkbox v-if="wasReplaced(orgPerson)"
+                  class="confirm-documents-checkbox mt-0 pt-0 mb-6"
+                  hide-details
+                  label="I confirm that the supporting documents have been received by the BC Registries."
+                  :rules="confirmDocumentsRules"
+                  v-model="orgPerson.confirmDocuments"
+                />
 
-                  <v-card v-if="wasBusinessSelectedFromLookup" outlined class="message-box rounded-0 mt-6">
-                    <p>
-                      <strong>Important:</strong> If the addresses shown below are out of date, you may
-                      update them here. The updates are applicable only to this registration.
-                    </p>
-                  </v-card>
+                <!-- Add firm org using look-up -->
+                <template v-if="showAddFirmOrgComponents && orgPerson.isLookupBusiness">
```

Comment:
I added the template element so the article section would be nested, like the other articles below. This is makes it nice and consistent when code-folding.

Reply:
PS Hide whitespace changes to review the code below (most of which hasn't changed other than indenting).